contributing/code/maintenance.rst

```diff
@@ -0,0 +1,58 @@
+Maintenance
+===========
+
+During the lifetime of a minor version, new releases (patch versions) are
+published on a monthly basis. This document describes the boundaries of
+acceptable changes.
+
```

Comment:
I suggest adding a general statement like:
This policy has one main goal: allow Symfony to keep getting better over time while avoiding the risk of breaking an existing and working application.